tools/slang-test/README.md

```diff
@@ -0,0 +1,84 @@
+# Slang Test
```

Comment:
Thanks for writing this up. This is really nice to have, and probably would have made life easier for you if it existed when you were diving into this code.

It would be great if we strive to have every major tool or sub-module have a decent `README.md` that orients a developer to what they should look out for when working with a corner of the codebase.